packages/webapp/socket_file_tests.js

```
@@ -85,6 +85,15 @@ Tinytest.add('socket file - remove socket file on exit', test => {
   });
 });
 
+Tinytest.add('socket file - no duplicate handlers on repeated registration', test => {
```

ESLint

[error] Replace `'socket·file·-·no·duplicate·handlers·on·repeated·registration',` with `⏎··'socket·file·-·no·duplicate·handlers·on·repeated·registration',⏎·`

(prettier/prettier)